lib/assets/javascripts/views/plans/new.js

```diff
@@ -49,14 +62,14 @@ $(() => {
       // Clear out the old template dropdown contents
       $('#plan_template_id option').remove();
 
-      // Fetch the available templates fbased on the funder and research org selected
+      // Fetch the available templates based on the funder and research org selected
       const jQueryForm = $('form#new_plan');
       const formElements = jQueryForm.serializeArray();
       $.ajax({
-        method: getMethod(jQueryForm),
-        url: getAction(jQueryForm),
+        method: 'POST',
```

Comment:
why are you hardcoding the method, could not you get it from the form method?

Reply:
Its not using the form's `action` in this case so I was reluctant to assume that the method would be the same.